demo/cwp-jdk11/README.md

```diff
@@ -0,0 +1,16 @@
+Jenkinsfile Runner demo
+===
+
+This demo demonstrates building of Jenkinsfile Runner Docker images
+with [Custom WAR Packager](https://github.com/jenkinsci/custom-war-packager/)
+using a JDK 11 based Docker image.
+This demo bundles some plugins so that it demonstrates a Pipeline execution.
+
+To build the Docker image, run `make clean build`
```

Comment:
Or just `make` since you made `all` an alias for those two targets.

Reply:
@jvz is right. We could refer to both in the README

Reply:
I will take this into account for future changes. I don't want to change it now because I want master and jdk11 branches not to differ except on the strictly necessary stuff.